MetaMask/metamask-extension
# Add Ink Networks

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/29368?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Add Ink network according to (network information)[https://docs.inkonchain.com/general/network-information] and ensure ink.svg is displayed properly
2. Add Ink Sepolia network according to (network information)[https://docs.inkonchain.com/general/network-information] and ensure ink-sepolia.svg is supported 

## **Screenshots/Recordings**



### **Before**
![Screenshot 2024-12-19 at 3 35 46 PM](https://github.com/user-attachments/assets/4ba64987-ba8b-49c4-af10-4c0e9f3e1e4a)



### **After**


![Screenshot 2024-12-19 at 3 35 12 PM](https://github.com/user-attachments/assets/cd27a1ed-cbd5-4206-8431-a759bffc6002)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
